Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalHBHEMuonFilter_Output_cff.py

```diff
@@ -0,0 +1,29 @@
+import FWCore.ParameterSet.Config as cms
+
+# output block for alcastream HCAL HBHEMuon
+# output module 
+#  module alcastreamHcalHBHEMuonOutput = PoolOutputModule
+OutALCARECOHcalCalHBHEMuonFilter_noDrop = cms.PSet(
+    # use this in case of filter available
+    SelectEvents = cms.untracked.PSet(
+        SelectEvents = cms.vstring('pathALCARECOHcalHBHEMuonFilter')
+        ),
+    outputCommands = cms.untracked.vstring( 
+        'keep *_hbhereco_*_*',
+        'keep *_ecalRecHit_*_*',
+        'keep *_offlineBeamSpot_*_*',
+        'keep *_hltTriggerSummaryAOD_*_*',
+        'keep *_TriggerResults_*_*',
+        'keep *_generalTracks_*_*',
```

Comment:
@davidlange6 here: https://indico.cern.ch/event/558794/contributions/2254652/attachments/1315042/1969666/Phase1_13.pdf.
The data throughput of this stream as it is now is large (see https://github.com/cms-sw/cmssw/pull/15266#issuecomment-234986136), but I understand that @bsunanda et al are working to reduce the rates. Also, as in the current version, the ALCARECO cannot be run (not propagated to /Configuration/etc./etc./